desc/compute/_field.py

```diff
@@ -2753,3 +2782,60 @@ def _B_dot_gradB_z(params, transforms, profiles, data, **kwargs):
         + data["B^zeta"] * data["|B|_zz"]
     )
     return data
+
+
+@register_compute_fun(
+    name="kappa",
+    label="\\kappa",
+    units="m^{-1}",
+    units_long="Inverse meters",
+    description="Curvature vector of magnetic field lines",
+    dim=3,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["J", "|B|", "b", "grad(|B|)"],
+)
+def _kappa(params, transforms, profiles, data, **kwargs):
+    data["kappa"] = -(
+        cross(data["b"], mu_0 * data["J"] + cross(data["b"], data["grad(|B|)"])).T
+        / data["|B|"]
+    ).T
+    return data
+
+
+@register_compute_fun(
+    name="kappa_n",
+    label="\\kappa_n",
```

Comment:
Would it be more clear to use `"\\kappa \\cdot \\hat{n}"`?

Reply:
This seems to be fairly standard notation in books on differential geometry, and its also what's used in Helander 2014